Source/Shared/BPConfiguration.h

```diff
@@ -78,6 +80,10 @@ typedef NS_ENUM(NSInteger, BPProgram) {
 @property (nonatomic, strong) NSNumber *launchTimeout;
 @property (nonatomic, strong) NSNumber *deleteTimeout;
 
+@property (nonatomic, strong) NSArray<NSString *> *commandLineArguments; // command line arguments for the app
```

Comment:
That comes from the xctestrun file and gets passed down to the individual `bp` processes